src/main/resources/liquibase/notification/changelog_1.0.xml

```diff
@@ -0,0 +1,24 @@
+<databaseChangeLog
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd">
+    <changeSet author="Dziyana Bahdanava" id="6">
```

Comment:
Usually we tell what's happening in this change set in ID field. Example:
`Add new Status column`. `Create Users table`. `Change name field type`